scripts/readme-gen/templates/README.tmpl.rst

```diff
@@ -0,0 +1,60 @@
+.. This file is automatically generated. Do not edit this file directly.
```

Comment:
Possibly add a jinja2 comment that's like:

```django
{# The following line is a lie. BUT! Once jinja2 is done with it, it will become truth! #}
```